src/pages/SplashPage.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../css/SplashPage.css';
+import SplashImage from '../img/land_image.jpg';
+
+function SplashPage() {
+  const SplashImageStyle = {
+    backgroundImage: `url(${SplashImage})`,
+    backgroundSize: 'contain',
+    backgroundPosition: 'center',
```

Comment:
Superb styling, but it would be great if in-line could be avoided and all are placed in the imported/linked styling page.

Reply:
Thanks. Noted.